core/base/src/TStyle.cxx

```diff
@@ -1527,6 +1527,9 @@ void TStyle::SetOptDate(Int_t optdate)
 ///  -  When "v"=2 all parameters are shown.
 ///
 ///  #### Notes:
+///
+///  - never call `SetOptFit(000111);` but `SetOptFit(1111)`, 0001111 will
```

Comment:
000111 != 1111

Reply:
Hey ! well spotted ... SetOptStat is wrong too in that case. Gratias !